src/components/accounts/settings/settings.css

```diff
@@ -0,0 +1,6 @@
+.container {
```

Comment:
Can you rename to settings-container or you can use the utility classes from here [bootstrap 5](https://getbootstrap.com/docs/5.0/utilities/flex/)

Reply:
I used the utility classes on bootstrap